index.html

```diff
@@ -2326,6 +2327,101 @@ <h2>Definition of Roles</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="comment">
+			<rdef>comment</rdef>
+			<div class="role-description">
+				<p>A comment contains a remark expressing a reaction to other content within the page.</p>
+				<p>The relationship of comments to other content SHOULD be exposed:</p>
+				<ol>
+					<li>When the comment refers to other content within the page, the source content SHOULD use <pref>aria-details</pref> to point to the related comment.</li>
+					<li>When the comment is a reply to another comment:
+						<ul>
+							<li>If possible, the reply comment SHOULD be a semantic descendant of the comment it relates to, either as a DOM descendant element, or via <pref>aria-owns</pref>. In this case, user agents MUST automatically compute the level and expose it to assistive technologies. This is the preferred method for defining
+								the comment hierarchy, as it creates a single root comment that can be referred to by commented content via <pref>aria-details</pref>.</li>
+							<li>Alternatively, when the parent comment is not rendered in the page, such as when comments are paginated, the hierarchical
+								level MAY be indicated via <pref>aria-level</pref>.</li>
+						</ul>
+					</li>
+				</ol>
+			</div>
+			<table class="role-features">
+				<caption>Characteristics:</caption>
+				<thead>
+					<tr>
+						<th scope="col">Characteristic</th>
+						<th scope="col">Value</th>
+					</tr>
+				</thead>
+				<tbody>
+					<tr>
+						<th class="role-abstract-head" scope="row">Is Abstract:</th>
+						<td class="role-abstract"> </td>
+					</tr>
+					<tr>
+						<th class="role-parent-head" scope="row">Superclass Role:</th>
+						<td class="role-parent"><rref>section</rref></td>
+					</tr>
+					<tr>
+						<th class="role-base-head" scope="row">Base Concept:</th>
+						<td class="role-base"> </td>
+					</tr>
+					<tr>
+						<th class="role-related-head" scope="row">Related Concepts:</th>
+						<td class="role-related"> </td>
+					</tr>
+					<tr>
+						<th class="role-scope-head" scope="row">Required Context Role:</th>
+						<td class="role-scope"> </td>
+					</tr>
+					<tr>
+						<th class="role-mustcontain-head" scope="row">Required Owned Elements:</th>
+						<td class="role-mustcontain">
+						</td>
```

Comment:
The `td` needs to be all on one line in order to be ignored by the script, i.e.
`<td class="role-mustcontain"> </td>`

(This will hide the empty "Required Owned Elements:" characteristic row - which is currently visible).

Reply:
Thanks!